### ISSUE TITLE
Feature: update $TABLE with a certain format

### ISSUE THREAD

**Issue by kylebaron** [metrumresearchgroup/bbr#710]
# Summary 

By default, NONMEM only includes 5 digits for numbers written to `.tab` output. This is usually ok for floating point numbers, we might want more and we need more when you have integer data with more than 5 digits (like record number data in a data set with > 99,999 records. 

Can we include a feature when we have a user friendly way to update `$TABLE` for 6 or more digits? I'd suggest letting users pass a `FORMAT` statement through to the control stream, but the syntax is difficult to remember (`FORMAT=s1PE12.5`).

**Comment by seth127** [comment]
Yes, we can definitely make this modification, and it shouldn't be too hard with `nmrec`.

My question is how we would present this to the user. Below are a few options off the top of my head. (Any of these would be controlled by an `T/F` argument, probably defaulting to `TRUE`.)

1. An argument in `copy_model_from()` (and maybe `new_model()`) that automatically adds this to the new control stream. _--> This feels a little aggressive to me, but if we really think it should be in there, it's probably the easiest_
2 A helper that adds this and could be piped from `copy_model_from()` or `new_model()`, similar to the [`tweak_initial_estimates()` examples](https://metrumresearchgroup.github.io/bbr/reference/tweak_initial_estimates.html#examples)
3. Part of a `check_model_file()` sort of helper, that a user would run before submitting. I'm envisioning this could also do things like check for the `NUM` column that `nm_join()` expects, and probably a few other conventions. This could also have a mode to warn/error or a mode to actually make the fix and message.
